### ISSUE TITLE
Navigation bar problem on Android

### ISSUE THREAD

**Issue by intel8** [creativetimofficial/argon-dashboard#2]
Navigation bar doesn't work on Android 4.4 and 5.1 stock browser or in webview. 
Menu not showing when tap on the hamburger menu.

**Comment by extrabright** [comment]
Hi,

I tested it and it seems to work well. I will try testing it on other Andorid browsers.

**Comment by carlospetrucio** [comment]
Yes the navbar opens but does not close on any mobile, and when it closes it takes 3 clicks.

**Comment by alagiesellu** [comment]
> Yes the navbar opens but does not close on any mobile, and when it closes it takes 3 clicks.

Yes and this need to be fixed.

**Comment by rarestoma** [comment]
Hi @carlospetrucio,

Please check this issue: https://github.com/creativetimofficial/argon-dashboard/issues/39. We also added it to the list for the next update.

I hope it helps.

All the best,
Rares

**Comment by carlospetrucio** [comment]
I managed to solve it, following his instructions.
Thank you very much friend.
